### DISCUSSION TITLE
Поддержка hentailib/mangalib

### DISCUSSION THREAD

**Discussion by ghost** [lirix360/ReadmangaGrabber#37] [General]
Безумно жаль, что МангаЛиб больше не поддерживается, но все же, можно рассмотреть возвращение его поддержки.
Кстати на hentailib доступна авторизация через почту и пароль

**Comment by lirix360** [comment]
В beta 7 скачивание с MangaLib возвращено в эксперементальном режиме.

**Reply by ghost** [reply]
Спасибки
